docs/en/setup/backend/backend-load-balancer.md

```diff
@@ -0,0 +1,9 @@
+# Backend Load Balancer
+
+At present, the Agent or Envoy service wants to exchange data with OAP, all of which are connected through direct connection, which will cause the problem of OAP load imbalance when exchanging large quantities.
+Skywalking-Satellite can be used as a Gateway, providing load balancing capabilities for data content before the data from Agent/Envoy reaches the OAP.
```

Comment:
```suggestion
Satellite is recommended to be used as a native gateway proxy, to provide load balancing capabilities for data content before the data from Agent/Envoy reaches the OAP. The major difference between Satellite and other general wide used proxy(s), like Envoy, is that, Satellite would routine the data accordingly to contents rather than connection, as gRPC streaming is used widely in SkyWalking.
```